migration/identity-migration/src/main/java/io/camunda/migration/identity/AuthorizationMigrationHandler.java

```diff
@@ -7,133 +7,155 @@
  */
 package io.camunda.migration.identity;
 
-import io.camunda.migration.identity.dto.UserResourceAuthorization;
+import static io.camunda.migration.identity.config.saas.StaticEntities.IDENTITY_DECISION_DEFINITION_RESOURCE_TYPE;
+import static io.camunda.migration.identity.config.saas.StaticEntities.IDENTITY_PROCESS_DEFINITION_RESOURCE_TYPE;
+
+import io.camunda.migration.identity.console.ConsoleClient;
+import io.camunda.migration.identity.console.ConsoleClient.Member;
+import io.camunda.migration.identity.dto.Authorization;
 import io.camunda.migration.identity.midentity.ManagementIdentityClient;
 import io.camunda.security.auth.Authentication;
 import io.camunda.service.AuthorizationServices;
+import io.camunda.service.AuthorizationServices.CreateAuthorizationRequest;
+import io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType;
 import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
 import io.camunda.zeebe.protocol.record.value.PermissionType;
-import java.util.AbstractMap.SimpleEntry;
-import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
-import java.util.Map.Entry;
 import java.util.Set;
 import java.util.stream.Collectors;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class AuthorizationMigrationHandler extends MigrationHandler<UserResourceAuthorization> {
 
-  private static final Logger LOG = LoggerFactory.getLogger(AuthorizationMigrationHandler.class);
+public class AuthorizationMigrationHandler extends MigrationHandler<Authorization> {
   private final AuthorizationServices authorizationService;
   private final ManagementIdentityClient managementIdentityClient;
+  private final ConsoleClient consoleClient;
 
   public AuthorizationMigrationHandler(
       final Authentication authentication,
       final AuthorizationServices authorizationService,
+      final ConsoleClient consoleClient,
       final ManagementIdentityClient managementIdentityClient) {
+    this.consoleClient = consoleClient;
     this.managementIdentityClient = managementIdentityClient;
     this.authorizationService = authorizationService.withAuthentication(authentication);
   }
 
-  // TODO: this will be revisited
   @Override
-  protected List<UserResourceAuthorization> fetchBatch(final int page) {
-    return managementIdentityClient.fetchUserResourceAuthorizations(SIZE);
+  protected List<Authorization> fetchBatch(final int page) {
+    // The authorizations are fetched without pagination
+    return List.of();
```

Comment:
🔧  comment doesn't fit the code, should we have `managementIdentityClient.fetchAuthorizations();` here?

Reply:
TIL as we don't have paging it has to return nothing otherwise it's called endlessly